eigentrust-cli/src/cli.rs

```diff
@@ -261,7 +290,61 @@ pub async fn handle_bandada(config: &ClientConfig, data: BandadaData) -> Result<
 	Ok(())
 }
 
-/// Handle `scores` and `local_scores` commands.
+/// Handles the deployment of AS contract.
+pub async fn handle_deploy(config: ClientConfig) -> Result<(), EigenError> {
+	let mnemonic = load_mnemonic();
+	let client = Client::new(config, mnemonic);
+	let as_address = deploy_as(client.get_signer()).await?;
+	info!("AttestationStation deployed at {:?}", as_address);
+
+	Ok(())
+}
+
+/// Handles proving key generation.
+pub fn handle_proving_key() -> Result<(), EigenError> {
+	let et_kzg_params = load_kzg_params(ET_PARAMS_K)?;
+	let proving_key = Client::generate_et_pk(et_kzg_params)?;
+
+	save_proving_key(proving_key)
+}
+
+/// Handles KZG parameters generation.
+pub fn handle_params(data: KZGParamsData) -> Result<(), EigenError> {
+	let pol_degree = data.k.as_ref().map_or(Ok(ET_PARAMS_K), |k| {
```

Comment:
Is it possible to make `k` mandatory?
It doesn't make sense to default to the k for EigenTrust circuit